dhis2-android-app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Why is necessary to add WRITE_EXTERNAL_STORAGE permission?

Reply:
Yes it was a mistake, sorry.